### ISSUE TITLE
Only show '[Process exited 139]'

### ISSUE THREAD

**Issue by XXiaoA** [ellisonleao/glow.nvim#65]
when i use :Glow it only show [Process exited 139]

**Comment by ellisonleao** [comment]
hey @XXiaoA can you give more information please?

**Comment by RaulOn18** [comment]
I'm having the same problem.
when i use `:Glow README.md` it only shows [Proccess exited 139]
![image](https://user-images.githubusercontent.com/48642800/161419957-c4003dfe-faba-4dfb-b2ec-79538e4d233b.png)

> ## my hardware specifications
*  Artix
* 5.16.16-artix1-1
* NVIM v0.6.1
> ## what I've accomplished

- [x] Install Glow With `:GlowInstall`
- [x] Reinstall Glow with `:PackerInstall`

**Comment by ellisonleao** [comment]
Are you guys able to run `glow` outside neovim?

**Comment by ellisonleao** [comment]
Guys can you pull latest and try again please? We did some improvements in the installation process which i hope can fix the issue on some distros

**Comment by XXiaoA** [comment]
> Guys can you pull latest and try again please? We did some improvements in the installation process which i hope can fix the issue on some distros

Yeah, I tried again just now, and it can work correctly. Thanks for ur great work!

**Comment by letscho123** [comment]
I have exactly the same issue.
If I type `glow` in the terminal I get `Segmentation fault`

Fedora 37 KDE
AstroNvim 2.11.2
Nvim 0.8.1
